### ISSUE TITLE
Map colours unclear (NA, low values, non-European countries)

### ISSUE THREAD

**Issue by doug-leasure** [OxfordDemSci/Mapineq#136]
The map in the dashboard uses the following colors which are very hard to distinguish from one another:
- white for NA, 
- grey for low values on the bivariate choropleth map, and 
- grey for countries outside of Europe.

Can we please revise the colours to avoid this confusion? This has long been on our list of fixes, and we continue to get feedback from users highlighting this issue.

**Comment by niekvanruler** [comment]
Because there is a difference between no data outside Europe and inside Europe @SjoerdGeodienst and I think it's better to keep the nodata values white.

**Comment by niekvanruler** [comment]
Shall we implement the black background ? @doug-leasure

**Comment by doug-leasure** [comment]
@niekvanruler , yes, I think we should. Does your team also prefer the dark basemap?

If we get mixed feedback from users, we could consider adding a toggle later to allow users to select light vs dark basemap.
